docs/application-connector/docs/041-environment-mapping-custom-resource.md

```diff
@@ -0,0 +1,32 @@
+---
+title: EnvironmentMapping
+type: Custom Resource
+---
+
+The `environmentmappings.remoteenvironment.kyma.cx` Custom Resource Definition (CRD) is a detailed description of the kind of data and the format used to enable APIs and Events from Remote Environment as a ServiceClass in a given Namespace. To get the up-to-date CRD and show the output in the yaml format, run this command:
```

Comment:
* from Remote Environment -> `from a Remote Environment`
* in the yaml format -> ``in the `yaml` format``